ui/app/templates/components/breadcrumb-trail.hbs

```diff
@@ -0,0 +1,9 @@
+<nav class="breadcrumb is-large">
+  <ul>
+    {{#each items as |item|}}
+      <li>
+        {{#link-to params=item.params data-test-breadcrumb=(get item.params "0")}}{{item.label}}{{/link-to}}
+      </li>
+    {{/each}}
+  </ul>
+</nav>
```

Comment:
One thing to note about components in Ember is that they are always wrapped by an element outside of the template. So where `breadcrumb-trail` is used, the resulting markup is actually

```html
<div>
  <nav class="breadcrumb is-large">
  ...
  </nav>
</div>
```

So maybe it's worth making that wrapper component the `nav` element by specifying `tagName` and `classNames` in the component's js file.

Reply:
Oh, another benefit of doing it this way is any classes given to the component will be placed on the nav element instead of the div:

```hbs
{{breadcrumb-trail items=breadcrumbs class="something-special-cased"}}
```